src/sentences/utils.rs

```diff
@@ -184,6 +184,22 @@ mod tests {
         assert_eq!(time.nanosecond(), 500_000_000);
     }
 
+    #[test]
+    #[should_panic]
+    fn test_parse_hms_fail() {
+        use chrono::Timelike;
+        let (_, time) = parse_hms("244519,").unwrap();
+        assert_eq!(time.hour(), 24);
+        assert_eq!(time.minute(), 45);
+        assert_eq!(time.second(), 19);
+        assert_eq!(time.nanosecond(), 0);
+        let (_, time) = parse_hms("244519.5,").unwrap();
+        assert_eq!(time.hour(), 24);
+        assert_eq!(time.minute(), 45);
+        assert_eq!(time.second(), 19);
+        assert_eq!(time.nanosecond(), 500_000_000);
+    }
+
```

Comment:
Would you mind removing this test and just leaving the RMZ in `all_supported_messages`.
I would suggest opening a new PR for the tests

Reply:
Ok I will do just that.